### PR TITLE
Adding Cemetech Presence

### PR DESCRIPTION
**Pull request by KnightsWhoSayNi0**
Hello! I am adding a presence for Cemetech.net (a calculator programming forum).
Here are some screenshots showing the code working:
homepage
topic
search

This imgur album has a bunch of the other things the presence displays.
[https://imgur.com/a/aupgR8i](https://imgur.com/a/aupgR8i)

### CODE REVIEW

File: websites/C/Cemetech/presence.ts [hunk 1]
```diff
@@ -0,0 +1,153 @@
+const presence = new Presence({
+  clientId: "829371771702345759" 
+});
+
+let browsingStamp = Math.floor(Date.now() / 1000), title: any;
```
Comment (EncryptedDev) [inline websites/C/Cemetech/presence.ts:5]: again, no point in letting this

File: websites/C/Cemetech/README.md [hunk 2]
```diff
@@ -0,0 +1,2 @@
+﻿# CemetechPreMid
```
Comment (EncryptedDev) [inline websites/C/Cemetech/README.md:1]: no readme

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Comment by EncryptedDev** [comment]
Wait I change my mind, you completely messed this up!! 
File structure has been broken & you even uploaded the `presence.js` lmao

**Comment by KnightsWhoSayNi0** [comment]
Sorry about that man. Fixed everything you said, hopefully it's all good now. :)

File: dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  "color": "#c11f1f",
+  "tags": [
+    "programming",
+    "forum"
+  ],
+  "url": "www.cemetech.net",
+  "description": {
+    "en": "Rich Presence for the Cemetech.net Calculator Programming Forum"
+  },
+  "settings": [
```
Comment (StrikerFRFX) [inline dist/metadata.json:21]: Settings cannot be blank, remove or add valid settings.

File: dist/presence.js [hunk 4]
```diff
@@ -0,0 +1,155 @@
+const presence = new Presence({
+    clientId: "829371771702345759"
+});
+let browsingStamp = Math.floor(Date.now() / 1000), title;
```
Comment (StrikerFRFX) [inline dist/presence.js:4]: Merge with previous const unless reassigned (which you don't)

**Comment by EncryptedDev** [comment]
remove these three files as they're dups of the other three files 
image

**Review by StrikerFRFX — CHANGES_REQUESTED** [review]
Hi @KnightsWhoSayNi0 , 
I've requested changes below in my review which you should take a look at 😄 
 -  In addition to changes listed, you fucked the files up somehow. For a start, `presence.ts` does not belong in the main dir, delete. 
 - Second of all, remove your `.js` files which should be in the `dist` folder upon compilation. The total file count should be 3, and yours is 6.

Final note, big yikes overall. If there are any more issues, we will continue once you fix what I have requested. If you cannot fix these, close your PR and make a new one from scratch that isn't facked.

Thanks,
**Striker**, PreMiD Reviewer Team.
Contact: me@strikerfrfx.dev, Striker#1337

**Comment by KnightsWhoSayNi0** [comment]
Will do. Not sure what happened to the root dir, but I'll fix it. I already removed the .js file but apparently it's still here? So sorry about all of this. I just did this project for fun and didn't expect the merge to be this big of a deal. I'll get everything fixed and get back to you soon. Thanks!

**Comment by StrikerFRFX** [comment]
> Will do. Not sure what happened to the root dir, but I'll fix it. I already removed the .js file but apparently it's still here? So sorry about all of this. I just did this project for fun and didn't expect the merge to be this big of a deal. I'll get everything fixed and get back to you soon. Thanks!

you may need to remove it using git as chances are you've just deleted the file as well, but git doesn't work that way. Or as I said, close this PR and open a new one with the fixed issues (and then just point to this PR for proof of working).

**Comment by KnightsWhoSayNi0** [comment]
I'll do that thanks. :)

File: dist/presence.js (hunk 4 above)
Comment (KnightsWhoSayNi0) [reply dist/presence.js]: getting compilation errors, will get back to this one.

**Comment by ririxidev** [comment]
Like I said before, presence and site itself is low quality.

**Comment by ririxidev** [comment]
> Wait I change my mind, you completely messed this up!! 
> File structure has been broken & you even uploaded the `presence.js` lmao

If you are going to tell everyone their issues, get out from here. Already told you that we don't need more help. I'm not going to tolerate your messages anymore.

**Pull request closed without merging**